packages/drivers/routerlicious-driver/package.json

```diff
@@ -128,6 +132,7 @@
 		"@fluidframework/protocol-definitions": "^3.2.0-237840",
 		"@fluidframework/server-services-client": "^3.1.0-240326",
 		"@fluidframework/telemetry-utils": "workspace:~",
+		"axios": "^1.6.2",
```

Comment:
Well, that is unsettling, but I am glad it has moved up.

Reply:
Hmm. Are there any runtime dependencies or are all of the imports type imports?
If these are only type imports, then I would have guessed it didn't need any special treatment. At least in other cases, I found that type imports appeared to work.

Reply:
The imports are `type import`. An eslint error prompted the adjustment, indicating that axios should be included in dependencies rather than devdeps. After running `git clean -xdf` locally, the error was resolved.

Reply:
If you were going off of eslint in VSCode, well it is a little untrustworthy. Restarting it helps with a lot of things. I think it gets some cached states and fails to update.
I tried to just use type imports directly from axios, but per the server packages that get involved things don't match. So, .cts works and why fight it?.